src/select.coffee

```diff
@@ -341,7 +342,15 @@ Ember.AddeparMixins.ResizeHandlerMixin,
 
   keyDown: (event) ->
     # show dropdown if dropdown is not already showing
-    return @set('showDropdown', yes) unless @get 'showDropdown'
+    # and the keycode should be one of special keys or common characters
+    # 'A'..'Z', 'a..z','0..9'
+    if event.keyCode in [@KEY_CODES.ENTER, @KEY_CODES.SPACEBAR] or
+    event.keyCode in [@KEY_CODES.DOWN, @KEY_CODES.UP] or
```

Comment:
@azirbel Do we have a style recommendation for something like this? Indenting subsequent lines of the condition means that the body of the conditional needs to be indented to 4 spaces, which is also weird.